### PR TITLE
fix(amplify-provider-awscloudformation): fix redudant upload message 

### LINKED ISSUE #5393: "amplify push" causes a weird "Uploading files" prompt
>**Note**: If your issue/bug is regarding the AWS Amplify Console service, please log it in the 
[Amplify Console GitHub Issue Tracker](https://github.com/aws-amplify/amplify-console/issues)

**Describe the bug**
Run "amplify push" in a new project. Observe how the "Uploading files..." is weirdly formatted. Expected: A "spinner" with Uploading files... in-place instead of consecutively printed.
image

**Amplify CLI Version**
You can use `amplify -v` to check the amplify cli version on your system
4.29.2

**Desktop (please complete the following information):**
 - OS: Mac
 - Node Version. You can use `node -v` to check the node version on your system

**Additional context**

### PR DESCRIPTION
**Pull request by yuth**
Update the s3 util accept a prameter which disables printing upload message. Updated appsync file
upload utilt to print upload message only once. Migrated S3 class to typescript

fix #5393

### CODE REVIEW

File: .circleci/config.base.yml [hunk 1]
```diff
@@ -694,6 +695,7 @@ workflows:
               only:
                 - master
                 - graphqlschemae2e
+                - fix-5393
```
Comment (ammarkarachi) [inline .circleci/config.base.yml:698]: @yuth I think you forgot to exclude this after your tests.

Comment (yuth) [reply .circleci/config.base.yml:698]: I am still running the tests. I will remove this when the tests pass

**Review by attilah — APPROVED** [review]
LGTM

**Comment by r0zar** [comment]
:tada:

**Pull request merged**